packages/treasury/src/stablecoinMachine.js

```diff
@@ -4,74 +4,45 @@ import '@agoric/zoe/exported.js';
 import '@agoric/zoe/src/contracts/exported.js';
 
 // The StableCoinMachine owns a number of VaultManagers, and a mint for the
-// "RUN" stablecoin. This overarching SCM will hold ownershipTokens in the
-// individual per-type vaultManagers.
+// "RUN" stablecoin.
 //
-// makeAddTypeInvitation is a closely held method that adds a brand new
-// collateral type. It specifies the initial exchange rate for that type.
-//
-// a second closely held method (not implemented yet) would add collateral of a
-// type for which there is an existing pool. It gets the current price from the
-// pool.
-//
-// ownershipTokens for vaultManagers entitle holders to distributions, but you
-// can't redeem them outright, that would drain the utility from the economy.
+// addVaultType is a closely held method that adds a brand new collateral type.
+// It specifies the initial exchange rate for that type. It depends on a
+// separately specified AMM to provide the ability to liquidate loans that are
+// in arrears. We could check that the AMM has sufficient liquidity, but for the
+// moment leave that to those participating in the governance process for adding
+// new collateral type to ascertain.
```

Comment:
Governance notes like this are good; too good to bury in the source code, in fact. Do you have a documentation update planned after this PR? Or do you see another path to get this info to the relevant audience?

"for the moment" smells like a TODO; is changing this critical? If not, perhaps take "for the moment out"?